src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: $widthCalendar;
+  &__day {
+    width: $widthDay;
+    height: $heightDay;
+    margin: $marginBetweenDays / 2;
+    box-sizing: border-box;
+    background-color: $backgroundDay;
+    border: 1px solid #000;
+    position: relative;
+    transition: all 0.5s ease 0s;
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
```

Comment:
The transform property in the &:hover pseudo-class is using translateY(-20px). Ensure that this transformation aligns with your design requirements, as it will move the element upwards by 20px on hover.